### PR TITLE
Greek keyboard update: add more diacritics (polytonic)

### LINKED ISSUE #7696: Greek Polytonic keyboard
If possible, please add Greek Polytonic as a language option for keyboard. It means that "Greek Extended" block (U+1F00 to U+1FFF) of Unicode will be added.

Thank you.

### PR DESCRIPTION
**Pull request by poire-z**
Provided by @ichnilatis-gr at https://github.com/koreader/koreader/issues/7696#issuecomment-864951373
Closes #7696.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7906)

### CODE REVIEW

File: frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua [hunk 1]
```diff
@@ -246,15 +325,25 @@ return {
     _Y_el = {
         "Υ",
         northeast = "Ύ",
-        northwest = "Y",
-        north = "Ϋ",
+        north = "Y",
+        west = "Ὑ",
+        south = "Ϋ",
+        southeast = "Ὕ",
+        southwest = "Ὗ",
```
Comment (poire-z) [inline frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua:332]: @ichnilatis-gr : you had these both associated to `southwest`. I changed one to `southeast`.
Please check as I don't know which of these feels best on the left or on the right.

Greek Capital Letter Upsilon _with Dasia and Oxia_ or _with Dasia and Perispomeni_ (with basil and pepperoni ? :) a bit hard to stomach though :)

Comment (ichnilatis-gr) [reply frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua:332]: Thank you for your notice.
However, I would prefer the reverse, to match the order of lowercase letters (as far as _Upsilon with Dasia and Oxia_ is concerned :pizza:).
```suggestion
        southwest = "Ὕ",
        southeast = "Ὗ",
```

Comment (poire-z) [reply frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua]: ok, done

**Review by Frenzie — APPROVED** [review]
lgtm!

File: frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua (hunk 1 above)
Comment (ichnilatis-gr) [reply frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua]: 👍

File: frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua [hunk 2]
```diff
@@ -246,15 +325,25 @@ return {
...
         "υ",
         northeast = "ύ",
-        northwest = "Y",
-        north = "ΰ",
-        west = "ϋ",
+        north = "y",
+        northwest = "ὺ",
+        east = "ὐ",
+        west = "ὑ",
+        south = "ΰ",
```
Comment (ichnilatis-gr) [inline frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua:341]: I'm sorry, but I've noticed some lines that need to be changed.

The combination "upsilon with acute and diaeresis" (ΰ) there is twice. So line 341 should be deleted
`south = "ΰ",`

**Comment by ichnilatis-gr** [comment]
I'm sorry, but I've noticed some lines that need to be changed.

The combination "upsilon with acute and diaeresis" (ΰ) there is twice. So line 341 should be deleted
`south = "ΰ",`

Please, alter line 128 to northwest, i.e. `northwest = "Ϊ",`
and line 330 to northwest too, i.e. `northwest = "Ϋ",`
This is for more comfort and consistency.

I'm sorry I don't know to use hithub efficiently.

**Comment by poire-z** [comment]
For these kind of small edits, you can use Github to edit files.
I let @hius07 give you some advices if needed, because that's how he does his PRs :)

**Comment by ichnilatis-gr** [comment]
I can't find the way I can edit the file.
It says "I must be on a branch to make or propose changes". What does this mean? How can I open the file to edit it?

(I know that the Greek "weed" bothered you a lot today, but if you could help me with this, please...)

**Comment by Frenzie** [comment]
I don't know how you ended up on not a branch exactly, but if you navigate from the front page you'll always be on the master branch.

https://github.com/koreader/koreader/blob/master/frontend/ui/data/keyboardlayouts/el_keyboard.lua

**Comment by ichnilatis-gr** [comment]
I think I just managed to make my corrections-proposes... :)

**Comment by poire-z** [comment]
So, you have edited the file and have a new branch https://github.com/ichnilatis-gr/koreader/tree/patch-1 - which looks ok.
Now, you need to propose it as a Pull-Request (dunno what kind of buttons you have to do that :)

**Comment by jimman2003** [comment]
Done
P.S there is a contribute button on the top of the branch that diviates from master

**Pull request merged**